src/vivarium/framework/results/context.py

```diff
@@ -138,14 +161,16 @@ def _format(field, param):
             return f"{str(field).upper()}_{param}"
 
         for params, val in data.iteritems():
+            if isinstance(params, str):  # handle single stratification case
```

Comment:
Call this `categories` as well

Reply:
That doesn't make sense for all the things, does it? (the measure and additional keys)?